src/GitLabHealth-Model-Importer/GitlabModelImporter.class.st

```diff
@@ -607,20 +602,34 @@ GitlabModelImporter >> importCommit: aCommitID ofProject: aGLHProject [
 	(self glhModel allWithType: GLHCommit) asOrderedCollection
 		detect: [ :commit | commit id = aCommitID ]
 		ifFound: [ :commit | ^ commit ].
-	result := self repoApi commits get: aCommitID inProject: aGLHProject id.
-	
+
+	result := self repoApi commits
+		          get: aCommitID
+		          inProject: aGLHProject id.
+
 	parsedResult := self parseCommitResult: result.
-	
-	parsedResult := self glhModel add: parsedResult unless: self blockOnIdEquality.
-	parsedResult := aGLHProject repository commits add: parsedResult unless: self blockOnIdEquality.
-		
+
+	parsedResult := self glhModel
+		                add: parsedResult
+		                unless: self blockOnIdEquality.
+	parsedResult := aGLHProject repository commits
+		                add: parsedResult
+		                unless: self blockOnIdEquality.
+
+	self withCommitDiffs ifTrue: [ self importDiffOfCommit: parsedResult ].
+
 	^ parsedResult
 ]
 
 { #category : #'import - commits' }
 GitlabModelImporter >> importCommitOfProject: anProject withId: anID [
 
 	| commit result |
+self
```

Comment:
I'm not sure but it seems that the deprecated method isn't calling the replaced one. Thus the old deprecated code is still use. Shouldn't it be replace for maintenability purpose ?